js/src/whalesong/chat.js

```diff
@@ -193,6 +193,11 @@ export class ChatManager extends ModelManager {
     );
   }
 
+  @command
+  async leaveGroup() {
+    return await (this.model.sendExit() && this.model.sendDelete());
```

Comment:
I think we should implement sendExit and sendDelete independently. Javascript code must avoid any logic. It is just an interface. Do you know what do each method? Do we really need sendDelete in order to leave group? Or it is just to remove chat?

Reply:
This is really a good point. I will make the command sendExit send Delete independently. With some tests I found that with sendExit the user exits the group(which should be this command) and with sendDelete the empty chat was deleted from both phone and WebWhatsApp.

With sendDelete in another command, the user will be able do delete any kind of chat and sendExit should be able to keep a record of the groups for some purpose.
I'm new in development and to contribute to open source, so I really appreciate the reviews. Thanks!